omod/src/main/webapp/fragments/field/datetimepicker.gsp

```diff
@@ -87,8 +87,9 @@
         minuteStep = 5
     }
     if(!datePickerFormat){
-        datePickerFormat = useTime ? "dd M yyyy hh:ii" : "dd M yyyy"
+        datePickerFormat = useTime ? "dd M yyyy HH:mm" :"dd M yyyy"
```

Comment:
Hi @dkayiwa 

This issue was found after the development mention on the issue. 

And it goes like this:
- Save a visit using dateTimePicker (usetime = false)
- Open said visit with dateTimePicker  (usetime = true)
- The dateTimePicker  instead of showing the correct date might mix days with months (depending on the initial date)